babel.config.js

```diff
@@ -5,7 +5,6 @@ module.exports = {
     `@babel/plugin-proposal-nullish-coalescing-operator`,
     [`@babel/plugin-proposal-decorators`, {legacy: true}],
     [`@babel/plugin-proposal-class-properties`, {loose: true}],
-    `@babel/plugin-proposal-async-generator-functions`,
```

Comment:
Node supports this by default https://node.green/#ES2018-features-Asynchronous-Iterators-async-generators